lib/Elastica/Search.php

```diff
@@ -523,6 +538,15 @@ public function setSuggest(Suggest $suggest)
     }
 
     /**
+     * @param boolean $onlySuggest
+     */
+    public function onlySuggest($onlySuggest = false)
```

Comment:
Maybe better to have by default $onlySuggest = true?

When I will try to call for example $search->onlySuggest() - I expected that onlySuggest will be enabled, but not disabled. It is how code will be readed.